ci-operator/step-registry/etcd-metrics-dump-service/etcd-metrics-dump-service-commands.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Injecting manifest to deploy etcd metrics dump service MachineConfig with script and systemd units"
+
+echo
+
+# Echo a script to be run as our systemd unit to a file so we can base64 encode it.
+sysd_script=$(cat << EOF
+#!/bin/bash
+. /etc/kubernetes/static-pod-resources/etcd-certs/configmaps/etcd-scripts/etcd.env
+curl -kLq --cacert \${ETCDCTL_CACERT//static-pod-certs/static-pod-resources/etcd-certs} \
+    --cert \${ETCDCTL_CERT//static-pod-certs/static-pod-resources/etcd-certs} \
+    --key \${ETCDCTL_KEY//static-pod-certs/static-pod-resources/etcd-certs} \
+    https://localhost:2379/metrics
```

Comment:
Should the output be filtered to only timeseries that are potentially interesting (if we know what those might be)?

Reply:
We're casting a wide net - all logs collected on all IPI installations for now - as we don't have a clear pattern on how to reproduce the issue. 

[Search.dptools](https://search.dptools.openshift.org/?search=operators+should+not+create+watch+channels+very+often&maxAge=48h&context=1&type=junit&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job) says its mostly affecting vsphere/gcp but its unclear how.